rendering_styles/default.render.xml

```diff
@@ -9665,8 +9682,8 @@
 					<apply_if nightMode="true" shader="quarry3_abandoned_night" color_0="#585858"/>
 				</case>
 				<apply_if showAccess="true" minzoom="13">
-					<switch>
-						<switch baseAppMode="car">
+					<switch rClass=".road.access">
+						<switch baseAppMode="car" rClass=".road.access.vehicle.$vehicle,.road.access.motor_vehicle.$motor_vehicle,.road.access.motorcar.$motorcar">
```

Comment:
this won't work as too complicated class.
You need to write here each class individuall
```
<case additional="vehicle=no" .road.access.motor_vehicle.no />
```